scipy/stats/tests/test_continuous.py

```diff
@@ -1031,19 +1031,25 @@ def test_shapes(self):
 
 
 class TestMakeDistribution:
-    @pytest.mark.parametrize('i, distdata', enumerate(distcont))
+    @pytest.mark.parametrize('i, distdata', enumerate(distcont + distdiscrete))
     def test_make_distribution(self, i, distdata):
         distname = distdata[0]
 
         slow = {'argus', 'exponpow', 'exponweib', 'genexpon', 'gompertz', 'halfgennorm',
                 'johnsonsb', 'kappa4', 'ksone', 'kstwo', 'kstwobign', 'powerlognorm',
-                'powernorm', 'recipinvgauss', 'studentized_range', 'vonmises_line'}
+                'powernorm', 'recipinvgauss', 'studentized_range', 'vonmises_line',
+                'betanbinom', 'zipf', 'logser', 'skellam'}  # discrete
         if not int(os.environ.get('SCIPY_XSLOW', '0')) and distname in slow:
             pytest.skip('Skipping as XSLOW')
 
-        if distname in {  # skip these distributions
-            'levy_stable',  # private methods seem to require >= 1d args
-            'vonmises',  # circular distribution; shouldn't work
+        if distname in {              # skip these distributions
+            'levy_stable',            # private methods seem to require >= 1d args
+            'vonmises',               # circular distribution; shouldn't work
+            'poisson_binom',          # vector shape parameter
+            'hypergeom',              # distribution functions need interpolation
```

Comment:
```python3
from scipy.special import _ufuncs as scu
M, n, N = 20, 7, 12
scu._hypergeom_pmf(3, n, N, M)  # np.float64(0.19865841073271417)
scu._hypergeom_pmf(4, n, N, M)  # np.float64(0.3575851393188855)
scu._hypergeom_pmf(3.5, n, N, M)  # np.float64(nan)
```
It would be helpful for the infinite sums if this would interpolate for non-integral arguments. We can add that, of course, but often this is something that happens naturally in the underlying implementation but is manually disabled. Any chance that's the case and we can enable it?

Reply:
The Boost Math implementation branches based on the size of `N`, and the large `N` implementation could be adapted to handle non-integral arguments, though it is currently constrained to unsigned values. 

https://github.com/boostorg/math/blob/a5c062542afb2037054c4e1b1be44db7b0859c38/include/boost/math/distributions/detail/hypergeometric_pdf.hpp#L467-L472

For the smaller `N` case, one coud evaluate the definition in terms of `binom`.

This is completely doable. I guess it's probably best to ask the Boost devs if they'd consider it.

Reply:
@mborland Some of the discrete distribution PMFs from Boost (e.g. binomial and negative binomial) are defined for non-integral values of the argument `k` whereas the hypergeometric distribution PMF returns NaN. There are some situations in which it is useful to approximate infinite sums involving the PDF with an integral, so if it is natural to define the hypergeometric distribution PMF for non-integral arguments, we'd appreciate it. Would you be willing/able to do that? If not, we can do our own interpolation; it would just be a bit slower. Thanks for considering it!

Reply:
I opened an [issue in math](https://github.com/boostorg/math/issues/1240) since this seems like a reasonable ask. I tagged you both so that you know when it's closed out.

Reply:
Thank you